.evergreen/csfle/activate_venv.sh

```diff
@@ -0,0 +1,25 @@
+set +x
+
+if [ "Windows_NT" = "$OS" ]; then
+  PYTHON_BINARY=C:/python/Python38/python.exe
+else
+  PYTHON_BINARY=python3
+fi
```

Comment:
Can you change this to:
```
if [ "Windows_NT" = "$OS" ]; then
  PYTHON_BINARY=C:/python/Python38/python.exe
elif [ command -v /opt/python/3.6/bin/python3 ]; then
  PYTHON_BINARY=/opt/python/3.6/bin/python3
else
  PYTHON_BINARY=python3
fi
```

I needed to do this to make this script work on RHEL 7.0:
```
$ python3
-bash: python3: command not found
$ ls /opt/python
2.4  2.5  2.6  2.7  3.2  3.3  3.4  3.5  3.6  jython2.5  jython2.7  pypy  pypy3.5  pypy3.6  toolchain_version
```

Reply:
Makes sense, RHEL always has different paths. Thanks for verifying on that distro; changed it to your version.